### ISSUE TITLE
Remove default for Lead Link (or anyone) filling unfilled roles?

### ISSUE THREAD

**Issue by brianjrobertson** [holacracyone/Holacracy-Constitution#186]
The default of Lead Link (or anyone else) filling unfilled roles sometimes creates a dynamic where people within the circle pile work they need from that role on the lead link, and then get frustrated when it's deprioritized, instead of volunteering for unfilled roles or helping find someone for them.  That brings up a question:  Do we need this default?  Is it helping at all?  Anyone have any use-cases where it's been important?

We could remove this default entirely, so unfilled roles are simply unfilled.  Instead, we could clarify the current default that a lead link holds undelegated circle-level accountabilities, so that any accountabilities on an unfilled role are still considered undelegated for requests coming from outside the circle.  Thus, the lead link would continue to hold any circle-level accountabilities to the outside world as-needed, so roles outside the circle could request projects/actions of the lead link based on those and the lead link couldn't duck their overall responsibility to hold the whole circle by claiming they're delegated to an unfilled role.  But, for requests coming from within the circle, the lead link could decline actions/projects due to an unfilled role existing, thus pushing the tension back to whoever has the request to try to get someone to fill the empty role...

Thoughts?

**Comment by tylerdanke** [comment]
If you remove Lead Link as filling unfilled roles, then you have situations where the Lead Link can assign to willing role fillers but doesn't. In those situationswho should be considered as filling that role?

**Comment by julianeroell** [comment]
I think the default is helpful. As a circle member, I want to be able to work within the given Role structure, which includes making requests to Roles. It is not my accountability to check if they are assigned or not. They exist in the structure, so I want to work with them.

If there were no Lead Link filling the roles by default, my requests would land in a "void", and there would be a risk of dumping important work on a "nonexisting" (unfilled) role, with work (or information) piling up there and nobody responsible.

With a Lead Link _basically_ in charge, things keep moving: When I have been in this situation as Lead Link - having unfilled roles on my circle, and requests coming in to them - I would usually ask the requestor if _they_ would be willing to fill the roles: and if only, to find a replacement / better partner to fill / outside help to get the work done. If they declined, then I could inform that I would hold the projects (as I my accountability), but would give no priority to them (otherwise I would have assigned myself to that role already). So this creates clarity for all, which would not be possible if the role were just left "empty", with no attention devoted to it.

TL;DR: I think the default of the Lead Link holding unfilled Roles of a circle is clear and helpful. (And thank you for making me think about it!)

**Comment by brianjrobertson** [comment]
Dropping this...
